packages/mobile/ios/Podfile

```diff
@@ -48,9 +48,12 @@ target 'AudiusReactNative' do
   pod 'RCT-Folly', :podspec => '../node_modules/react-native/third-party-podspecs/RCT-Folly.podspec'
   pod 'boost', :podspec => '../node_modules/react-native/third-party-podspecs/boost.podspec'
 
-  use_native_modules!
-  use_flipper!({ "Flipper-DoubleConversion" => "1.1.7" })
```

Comment:
this was causing a mysterious error (https://github.com/facebook/react-native/issues/32333) so i got rid of it since none of us use flipper